jest.config.js

```
@@ -17,6 +16,14 @@ module.exports = {
   ],
   coverageDirectory: 'coverage',
   coverageReporters: ['text', 'lcov', 'html'],
+  coverageThreshold: {
+    './src/services/transfer/': {
```

ESLint

[error] Replace `'./src/services/transfer/'` with `"./src/services/transfer/"`

(prettier/prettier)